src/main/java/thePackmaster/cards/anomalypack/SouleaterStrike.java

```diff
@@ -14,10 +14,11 @@ public class SouleaterStrike extends AbstractAnomalyCard {
     public static final String ID = SpireAnniversary5Mod.makeID("SouleaterStrike");
     private static final int COST = 1;
     private static final int ATTACK_DMG = 13;
+    private static final int MAGIC = 13;
 
     public SouleaterStrike() {
```

Comment:
The creator of the Anomaly pack made their own fix for this and ThoughtWeaving, so you'll need to resolve the merge conflicts here.